contracts/service-registry/src/state.rs

```diff
@@ -36,16 +36,24 @@ pub struct Worker {
     pub service_name: String,
 }
 
-impl TryInto<Participant> for Worker {
```

Comment:
Does clippy not catch this kind of stuff? `TryInto` should never have been implemented but `TryFrom`.

Reply:
Found this: https://rust-lang.github.io/rust-clippy/master/index.html#/from_over_into, but I did some tests and apparently clippy will catch `Into` but not `TryInto` lol